server.js

```diff
@@ -1,22 +1,21 @@
 (function(global) {
 
 const HERE_FILE = 'here.txt';
-const PASSWORD_FILE = 'password.txt';
-const SECRET_FILE = 'secret.txt';
 
 var app = require('express')();
 var server = require('http').Server(app);
+var port = process.env.PORT || 5000;
 var fs = require('fs');
 
 function read(file) {
   return fs.readFileSync(file, {encoding: 'utf8'}).trim();
 }
 
-const SECRET = read(SECRET_FILE);
+const SECRET = process.env.SECRET_MSG;
```

Comment:
Heroku environment variables aren't affected by reboots, but files are :-(. None of this will mater after the switch to firebase, though.